src/pages/home/report/ReactionList/PopoverReactionList/BasePopoverReactionList.tsx

```diff
@@ -21,13 +21,12 @@ function BasePopoverReactionList(props: BasePopoverReactionListPropsWithLocalWit
         preferredLocale,
     });
     // Get the reaction information
-    const {emojiCodes, reactionCount, hasUserReacted, users} = getReactionInformation();
-
+    const {emojiCodes, reactionCount, hasUserReacted, users, isReady} = getReactionInformation();
     useImperativeHandle(ref, () => ({hideReactionList, showReactionList}));
 
     return (
         <PopoverWithMeasuredContent
-            isVisible={isPopoverVisible}
+            isVisible={isPopoverVisible && isReady}
```

Comment:
I noticed that it's possible the other information isn't ready yet when the popover is visible, instead of checking each information one-by-one, I returned the `isReady` variable from `getReactionInformation`.
https://github.com/Expensify/App/assets/50919443/662d3504-8f3b-4caf-9eda-29375ac412de